src/main/java/uk/nhs/tis/sync/job/PostFundingStatusSyncJob.java

```diff
@@ -100,29 +100,32 @@ protected int convertData(Set<Post> entitiesToSave, List<EntityData> entityData,
       Post post = entityManager.find(Post.class, entity.getEntityId());
 
       if (post != null) {
-        Set<PostFunding> fundings = post.getFundings();
-
-        if (fundings.size() > 1) {
-          boolean allEndDatesYesterday = true;
-          for (PostFunding funding : fundings) {
-            if(!yesterday.equals(funding.getEndDate())) {
-              allEndDatesYesterday = false;
-              break;
-            }
-          }
-          if(allEndDatesYesterday) {
-            post.setFundingStatus(Status.INACTIVE);
-          }
-          entitiesToSave.add(post);
-        } else if (fundings.size() == 1) {
-          post.setFundingStatus(Status.INACTIVE);
-          entitiesToSave.add(post);
-        }
+        processPostFundingStatus(entitiesToSave, yesterday, post);
       }
     }
     return entities - entitiesToSave.size();
```

Comment:
The return value of this method is about how many records have been skipped. 
So, in your unit tests, if the field fundingStatus is not updated in the sync process, the record should be skpped, and return value should plus 1.

Reply:
Thanks for spotting this